cmd/tracetest/main.go

```diff
@@ -1,5 +1,7 @@
 package main
 
+// THIS FILE IS NOT UP TO DATE
```

Comment:
what's the purpose / plan for this file?

Reply:
This file was used to test tracer and can client but is now out of date with bus manager/tracer refactor. I think once can client is deleted/merged with bus manager, then this file will be deleted.